partials/home-splash.php

```diff
@@ -0,0 +1,12 @@
+<?php
+$home = get_page_by_path('home');
+$splash_text = get_post_meta($home->ID,'_igv_splash_text', true);
```

Comment:
but then again you have the conditional for the other partials so that works too it is just missing